packages/ui/src/views/repo/pull-request/components/reviewers/pull-request-reviewers-item.tsx

```diff
@@ -19,9 +19,11 @@ const ReviewerItem = ({ reviewer, reviewDecision, sha, sourceSHA, processReviewD
   }
   return (
     <div key={reviewer?.id} className="flex items-center justify-between space-x-2">
-      <div className="flex items-center space-x-2">
+      <div className="flex items-center space-x-2 overflow-hidden">
```

Comment:
this can probably be replaced by `LayoutV2.Horizontal` component. But not a blocker